docusaurus/docs/tutorials/ui-extensions/register-an-extension.md

```diff
@@ -0,0 +1,235 @@
+---
+id: register-an-extension
+title: Register an extension
+description: Make part of the UI extendable with links or components defined by plugins
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - extensions
+  - ui-extensions
+sidebar_position: 20
+---
+
+An "extension" is either a link or a React component defined by a plugin and rendered either somewhere in the core Grafana UI or in an other app plugin.
+[Read more about extensions under key concepts](../../key-concepts/ui-extensions.md).
```

Comment:
as in create an extension point

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/954/commits/999f5a2263de23d9f7fd41e5f47d07843fc71ed7.